tests/installation/ipxe_install.pm

```diff
@@ -357,6 +358,19 @@ sub run {
         return;
     }
 
+    if (get_var('WORKER_CLASS') =~ /ipmi-nvdimm/) {
+        assert_screen 'nue-ipxe-menu', 600;
+        if (match_has_tag("nue-ipxe-menu")) {
```

Comment:
I don't think this `if (match_has_tag(...))` is necessary. You're only asserting one tag, so either the test fails in L362, or the screen assertion succeeds on a needle that has `nue-ipxe-menu` as the tag.

Reply:
thank you, I updated the code.